docs/UserGuide.md

```diff
@@ -259,7 +338,11 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd company in the intern book.
```

Comment:
maybe can follow the mark and unmark feature to make it more consistent, perhaps can include listed intern book, and if we are putting list first, we should do the same for mark and unmark as well

Also, maybe we can add the error handling of the index, similar to the mark and unmark feature as well, where it states that if the index is out of bound, an error message will be displayed